File: app/src/main/java/com/zuk0/gaijinsmash/riderz/ui/activity/main/fragment/trip/TripFragmentModule.java
package com.zuk0.gaijinsmash.riderz.ui.activity.main.fragment.trip;

import dagger.Module;
import dagger.Provides;

@Module
public class TripFragmentModule {
    @Provides
    TripViewModel provideTripViewModel() {
        return new TripViewModel();
    }
}

File: app/src/main/java/com/zuk0/gaijinsmash/riderz/ui/activity/main/fragment/trip/TripFragmentModule.kt
package com.zuk0.gaijinsmash.riderz.ui.activity.main.fragment.trip

import dagger.Module
import dagger.Provides

@Module
class TripFragmentModule {
    @Provides
    fun provideTripViewModel(): TripViewModel {
        return TripViewModel()
    }
}